tools/clang/test/CodeGenSPIRV/rich.debug.type.member.function.hlsl

```diff
@@ -41,11 +41,10 @@ struct foo : base {
 // CHECK: [[set:%\d+]] = OpExtInstImport "OpenCL.DebugInfo.100"
 
 // CHECK: [[fooName:%\d+]] = OpString "foo"
-// CHECK: [[foo:%\d+]] = OpExtInst %void %1 DebugTypeComposite [[fooName]] Structure {{%\d+}} 22 1 {{%\d+}} [[fooName]] %uint_352 FlagIsProtected|FlagIsPrivate
+// CHECK: [[bool:%\d+]] = OpExtInst %void [[set]] DebugTypeBasic {{%\d+}} %uint_32 Boolean
+// CHECK: [[foo:%\d+]] = OpExtInst %void %1 DebugTypeComposite [[fooName]] Structure {{%\d+}} 22 8 {{%\d+}} [[fooName]] %uint_0 FlagIsProtected|FlagIsPrivate
```

Comment:
> why is the size param of the DebugTypeComposite uint_0 ?

As I mentioned, it is because the unknown physical layout.

> I know this hasn't been change by this PR, but why is the parent argument here fooName ? meaning fooName is the parent for fooName?

[DebugTypeComposite](https://www.khronos.org/registry/spir-v/specs/unified1/OpenCL.DebugInfo.100.html#DebugTypeComposite) has operands Name, Tag, Source, Line, Column,
Parent, Linkage Name, Size, Flags, Members.
`fooName` is used for Name and Linkage Name operands.
I just avoid specifying the Parent using `{{%\d+}}`.